app/controllers/api/projects.php

```diff
@@ -22,6 +22,7 @@
 use Utopia\Cache\Cache;
 use Utopia\Config\Config;
 use Utopia\Database\Database;
+use Utopia\Database\DateTime as DatabaseDateTime;
```

Comment:
```suggestion
use Utopia\Database\DateTime;
```
Can we just not use this ?

Reply:
Done.